Gopkg.lock

```diff
@@ -9,6 +9,29 @@
   revision = "fcb9a2d5f791d07be64506ab54434de65989d370"
   version = "v0.37.4"
 
+[[projects]]
```

Comment:
I think this file was also auto-generated?

Reply:
done